SSR wheel renders fail when postcss is not bundled

### Describe the bug

Current `main` builds start the Node SSR proxy, but every standalone SSR render returns HTTP 500 with `Cannot find module 'postcss'`. Gradio then times out waiting for a non-5xx Node response and leaves Python on its internal port.

`@gradio/sanitize` uses `sanitize-html`, whose style-attribute parsing loads `postcss`. Since the Vite 8 upgrade in #13329, `postcss` is externalized from the server bundle. The wheel's `gradio/templates/node/build` output installs only `http-proxy`, so `postcss` is unavailable at runtime. This is masked in the monorepo because the top-level `node_modules` contains `postcss`.

### Reproduction

1. Build the frontend from current `main`.
2. Copy `gradio/templates/node/build/server`, the generated build `package.json`, and the build's production `node_modules` to an isolated temporary directory.
3. Initialize the generated SvelteKit `Server` and render `/`.

The response is HTTP 500 and Node reports `Cannot find module 'postcss'`. A wheel-based Gradio app with `ssr_mode=True` exhibits the same startup failure on port 7860.

### Expected behavior

The shipped Node build should render successfully without relying on the repository's top-level dependencies. The frontend build should fail if its standalone SSR output references a runtime dependency that is not shipped.

### Additional context

Adding `postcss` to `ssr.noExternal` bundles it into the SSR output. With that change, the isolated render check passes and a local Gradio SSR app returns HTTP 200 from the Node proxy on port 7860.

--- a/.changeset/thin-donuts-make.md
+++ b/.changeset/thin-donuts-make.md
@@ -1,0 +1,5 @@
+---
+"@self/app": minor
+---
+
+feat:Bundle postcss in standalone SSR builds

--- a/js/app/package.json
+++ b/js/app/package.json
@@ -4,7 +4,7 @@
 	"private": true,
 	"scripts": {
 		"dev": "vite dev",
-		"build": "vite build && node after_build.js && node cleanup_build.js",
+		"build": "vite build && node after_build.js && node verify_build.js && node cleanup_build.js",
 		"preview": "vite preview",
 		"check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
 		"check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",

--- a/js/app/verify_build.js
+++ b/js/app/verify_build.js
@@ -1,0 +1,86 @@
+import {
+	cpSync,
+	existsSync,
+	mkdirSync,
+	mkdtempSync,
+	rmSync,
+	copyFileSync
+} from "fs";
+import { tmpdir } from "os";
+import { dirname, join, resolve } from "path";
+import { spawnSync } from "child_process";
+import { fileURLToPath, pathToFileURL } from "url";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+const out_path = resolve(__dirname, "../../gradio/templates/node/build");
+
+async function render(build_path) {
+	const { Server } = await import(
+		pathToFileURL(join(build_path, "server", "index.js")).href
+	);
+	const { manifest } = await import(
+		pathToFileURL(join(build_path, "server", "manifest.js")).href
+	);
+
+	const server = new Server(manifest);
+	await server.init({ env: {} });
+	const response = await server.respond(
+		new Request("http://127.0.0.1/", {
+			headers: { "x-gradio-server": "http://127.0.0.1:1" }
+		}),
+		{ getClientAddress: () => "127.0.0.1" }
+	);
+
+	if (response.status >= 500) {
+		const body = await response.text();
+		throw new Error(
+			`Standalone SSR render returned ${response.status}: ${body.slice(0, 500)}`
+		);
+	}
+}
+
+function verify_build() {
+	const temporary_directory = mkdtempSync(join(tmpdir(), "gradio-ssr-build-"));
+	const isolated_build = join(temporary_directory, "build");
+
+	try {
+		mkdirSync(isolated_build);
+		copyFileSync(
+			join(out_path, "package.json"),
+			join(isolated_build, "package.json")
+		);
+		cpSync(join(out_path, "server"), join(isolated_build, "server"), {
+			recursive: true
+		});
+		if (existsSync(join(out_path, "node_modules"))) {
+			cpSync(
+				join(out_path, "node_modules"),
+				join(isolated_build, "node_modules"),
+				{ recursive: true }
+			);
+		}
+
+		const result = spawnSync(
+			process.execPath,
+			[__filename, "--render", isolated_build],
+			{
+				cwd: isolated_build,
+				env: { ...process.env, NODE_PATH: "" },
+				stdio: "inherit"
+			}
+		);
+
+		if (result.status !== 0) {
+			throw new Error("Standalone SSR build verification failed");
+		}
+	} finally {
+		rmSync(temporary_directory, { recursive: true, force: true });
+	}
+}
+
+if (process.argv[2] === "--render") {
+	await render(process.argv[3]);
+} else {
+	verify_build();
+}

--- a/js/app/vite.config.ts
+++ b/js/app/vite.config.ts
@@ -45,7 +45,7 @@ export default defineConfig(({ mode }) => {
 			resolve: {
 				conditions: ["gradio"]
 			},
-			noExternal: ["@gradio/*", "@huggingface/space-header"],
+			noExternal: ["@gradio/*", "@huggingface/space-header", "postcss"],
 			external: mode === "development" ? [] : ["svelte", "svelte/*"]
 		},
 		build: {

